modules/liveyieldAnalyticsAdapter.js

```diff
@@ -0,0 +1,212 @@
+import adapter from 'src/AnalyticsAdapter';
+import adaptermanager from 'src/adaptermanager';
+import CONSTANTS from 'src/constants.json';
+import * as utils from 'src/utils';
```

Comment:
Due to a recent merge (see #3435), we now require any imports for a core or `modules` file (like this Analytics adapter file) to use a relative path style.

Can you please update these imports here?